src/pal/src/include/pal/stackstring.hpp

```diff
@@ -5,7 +5,9 @@
 #ifndef __STACKSTRING_H_
 #define __STACKSTRING_H_
 
+#ifndef __STACKSTRING_H_STANDLONE_
 #include "pal/malloc.hpp"
```

Comment:
I would rather remove the `#include "pal/malloc.hpp"` from here. There are 14 places in PAL that include the `stackstring.hpp` and only 5 of them are missing inclusion of the `malloc.hpp` before the `stackstring.hpp`. So it is easy to fix that.